modules/azure/api_connectors/log_analytics/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "resource_group_name" {
+  type        = string
+  description = "Location For the Connection"
+}
+
+variable "log_analytics_workspace_connection_name" {
```

Comment:
In the other files, it is called "connection_name". Lets do that here too.
Also, we may want validation similar too this?
validation {
    condition     = contains(["azuretables", "azureblob"], var.connection_api_name)
    error_message = "Argument \"connection_api_name\" must be either \"azuretables\" or \"azureblob\"."
  }

Reply:
That one is specifically for storage accounts because there is a difference between blob and table connectors. For log analytics it's only one.

Reply:
Ok, then let's keep it open and not force people to use certain names :)